chatwoot/chatwoot
# chore: Add `custom attributes` in `campaign.triggered` event

Fixes #4447 

**How to QA?**

1. Create a new conversation attribute
2. Enable custom attribute in pre-chat form settings
3. Open the widget through a campaign
4. Check the conversation attribute is updating after creating a new conversation
